src/styles/components/_NavAccordion.scss

```diff
@@ -0,0 +1,222 @@
+.NavAccordion {
+
+	@include font-heading;
+	@include text-sm;
+	list-style: none;
+
+	a:link,
+	a:visited {
+		color: inherit;
+	}
+
+	ul,
+	ol {
+		list-style: none;
+		font-size: inherit;
+		margin: 0;
+		position: relative;
+		transition: max-height ease-in-out 0.1s;
+	}
+
+}
+
+.NavAccordion_Item {
+
+	font-size: inherit;
+	display: block;
+	border-bottom: 1px solid $border-color;
+
+	&:last-child {
+		border-bottom: none;
+	}
+
+	.NavAccordion_Anchor {
+		display: block;
+		padding: #{ $base-line-height / 4 } 0;
+		position: relative;
+	}
+
+	> ul {
+		overflow: hidden;
+		margin-left: $gutter-width / 2;
+	}
+
+}
+
+.NavAccordion_Item > ul:after {
+	content: " ";
+	border-left: 1px solid $border-color;
+	top: -.333333333rem;
+	bottom: 1.111111rem;
+	left: 0;
+	position: absolute;
+}
+
+.NavAccordion_Item .NavAccordion_Item {
+
+	margin-left: 0;
+	padding-left: $gutter-width;
+	border-left: none;
+	border-bottom: none;
+
+	.NavAccordion_Anchor:after {
+		content: " ";
+		width: $gutter-width / 2;
+		border-bottom: 1px solid $border-color;
+		position: absolute;
+		top: 1.1rem;
+		left: $gutter-width * -1;
+	}
+
+	> ul {
+		overflow: visible;
+		margin-left: 0;
+	}
+
+}
+
+.NavAccordion_Toggle {
+
+	@extend .Btn-Small;
+	display: block;
+	position: absolute;
+	right: 0;
+	top: 0;
+	margin: 0;
+	padding: 0;
+	height: $base-line-height * 1.5;
+	width: $base-line-height * 1.5;
+	text-indent: 100%;
+	overflow: hidden;
+	background: transparent;
+
+	&:before {
+		position: absolute;
+		content: " ";
+		background: rgba( $hm-warm-grey, .75 );
+		border-radius: 100%;
+		top: 50%;
+		left: 50%;
+		width: 18px;
+		height: 18px;
+		margin-left: -9px;
+		margin-top: -9px;
+		text-indent: 0;
+	}
+
+	&:after {
+		position: absolute;
+		content: " ";
+		background: url( "#{ $hm-images }/icons/icon-plus-white.svg" ) no-repeat center center;
+		background-size: 100% 100%;
+		top: 50%;
+		left: 50%;
+		width: 8px;
+		height: 8px;
+		margin-left: -4px;
+		margin-top: -4px;
+		text-indent: 0;
+	}
+
+	&:hover,
+	&:focus {
+
+		background: none;
+		outline: none;
+
+		&:before {
+			background: darken( $hm-warm-grey, 40% );
+		}
+	}
+
+	.NavAccordion_Item-Open & {
+		&:after {
+			background-image: url( "#{ $hm-images }/icons/icon-minus-white.svg" );
+		}
+	}
+}
+
+@mixin NavAccordion_Coloured( $bg, $border-color, $text-color: $hm-warm-grey ) {
```

Comment:
Maybe at the top of the file? This is mostly just a workaround to avoid having to write it out twice. Its very specific to this component though so not keen on moving to a separate file.